### ISSUE TITLE
Selection and Pasting Issues

### ISSUE THREAD

**Issue by jonch070** [gwatcha/reaper-keys#76]
Pasting multiple items to tracks pastes them on last touched track even if that is part of a track selection (like if you visual track select from the top down it pastes things on the bottom track). Default reaper behaviour will be the best way around this perhaps, with pasting on topmost selected track?
![pasting issue](https://github.com/user-attachments/assets/18afb180-d960-48ce-a095-bf6026e06ac9)

It also seems to be the case that selecting only works when visual timeline mode is triggered after visual track, so you cannot select if you use visual timeline first. 
![selection issue](https://github.com/user-attachments/assets/2023fb14-730c-4316-8643-fa7eef4f257c)

I am also unable to run actions on the selected items like "cut" if I selected them with the mouse. Perhaps related
![delete issue](https://github.com/user-attachments/assets/19c98131-203c-4054-adad-ba5bc6fd55c7)

**Comment by molleweide** [comment]
yeah, a lot of work and improvements can be done on copy pasting, mostly because ig it is a bit complicated to its nature, there are so many ways it could be done and custom workflows. i have been doing custom some custom hacks on my own machine. everyting is solvable but it would maybe require a more feature complete yank, cut, put, mechanism analogous to how it works in eg vim/nvim with multiple types of registers and various manners for specifying how the information should be pasted.

**Comment by myrrc** [comment]
I can reproduce the bug and I'll try to fix it during next week
